Watchman.DomainModel/Watchman.DomainModel.csproj

```diff
@@ -6,8 +6,6 @@
 
   <ItemGroup>
     <Folder Include="Help\Commands\" />
-    <Folder Include="Settings\Commands\Handlers\" />
```

Comment:
usunąłeś to celowo?

Reply:
Z tego co mi się wydaje to VS usunął z pliku projektu bo wcześniej ten folder był pusty, a teraz dodałem klasy, więc już nie musi trzymać go w pliku projektu